prov/efa/src/rdm/rxr_pkt_type_data.c

```diff
@@ -78,13 +80,10 @@ int rxr_pkt_init_data(struct rxr_pkt_entry *pkt_entry,
 	/*
 	 * Data packets are sent in order so using bytes_sent is okay here.
 	 */
-	data_hdr->seg_offset = ope->bytes_sent;
-	data_hdr->seg_length = MIN(ope->total_len - ope->bytes_sent,
-				   ep->max_data_payload_size);
-	data_hdr->seg_length = MIN(data_hdr->seg_length, ope->window);
+	data_hdr->seg_offset = data_offset;
+	data_hdr->seg_length = data_size;
 	ret = rxr_pkt_init_data_from_ope(ep, pkt_entry, hdr_size,
-					      ope, ope->bytes_sent,
```

Comment:
Is `ope->bytes_sent` the same with `data_offset`?

Reply:
should be `data_offset` here. because we are initializing multiple data packets at the same time, before they are sent.